src/redux/actions/item.js

```diff
@@ -14,6 +14,16 @@ const add = ({ name, photo, soldCount, count, price }) => ({
   }
 });
 
+const sell = (id, item) => {
+  return dispatch => {
+    // async transaction calls here
+    item.soldCount += 1;
+    item.count -= 1;
```

Comment:
this logic goes to reducer

Reply:
don't think you need to pass in the item at all

Reply:
Might be selling multiple items at the same time. We should consider having a shopping cart